### ISSUE TITLE
Running vips on  openjdk:8-jre-alpine

### ISSUE THREAD

**Issue by jvence** [jcupitt/docker-builds#7]
Any chance this would compile and run on openjdk:8-jre-alpine? Tried modifying the amazonlinux2 dockerfile to not avail. Thanks

**Comment by jcupitt** [comment]
Sure, for example:

https://github.com/jcupitt/docker-builds/blob/master/libvips-openjdk-alpine/Dockerfile

If you need HEIC support it's a bit harder. You'd probably need to build libheif as well.

**Comment by jvence** [comment]
Yes I would need libheif. Can I modify https://github.com/jcupitt/docker-builds/blob/master/libvips-openjdk-alpine/Dockerfile to build libheif? Any pointers would be greatly appreciated.

**Comment by jcupitt** [comment]
libheif in turn depends on some other packages, so you'd need to check those first.

HEIC decode uses libde265 and encode uses libx265. They have patent issues, so you'll need to be extremely careful with them.

libheif also supports AVIF, which needs libaom and erm something else I forget. If AVIF is enough, you'll need them too.

This builds libheif and libx265:

https://github.com/jcupitt/docker-builds/blob/master/ruby-vips-amazonlinux/Dockerfile
